clients/fides-js/src/components/tcf/TcfOverlay.tsx

```diff
@@ -252,17 +271,34 @@ const TcfOverlay: FunctionComponent<OverlayProps> = ({
           </ConsentBanner>
         ) : null
       }
-      renderModalContent={({ onClose }) => (
-        <TcfModalContent
-          experience={experience}
-          draftIds={draftIds}
-          onChange={handleUpdateDraftState}
-          onSave={(keys) => {
-            handleUpdateAllPreferences(keys);
-            onClose();
-          }}
-        />
-      )}
+      renderModalContent={({ onClose }) => {
+        const onSave = (keys: EnabledIds) => {
+          handleUpdateAllPreferences(keys);
+          onClose();
+        };
+        return (
+          <div>
+            <TcfTabs
+              experience={experience}
+              enabledIds={draftIds}
+              onChange={handleUpdateDraftState}
+              activeTabIndex={activeTabIndex}
+              onTabChange={setActiveTabIndex}
+            />
+            <TcfConsentButtons
+              experience={experience}
+              onSave={onSave}
+              firstButton={
+                <Button
+                  buttonType={ButtonType.SECONDARY}
+                  label={experience.experience_config?.save_button_label}
```

Comment:
what's the default if there's no `save_ button_label` configured?

Reply:
oh hm I guess there isn't one. I'll check if we have a fallback elsewhere!

Reply:
it doesn't look like we have a fallback for "save" anywhere. do you think we should put one? I don't mind not having one since it'd be a reminder for the user they need to fill out their experience config if it shows up blank, but maybe that's not a good customer experience :) I can see it being a tricky thing to debug if it always says i.e. "Save" and a user expects it to say something else

Reply:
Looking a bit, I see that the BE _should_ always populate the labels with defaults in `fides/data/privacy_notices/privacy_experience_config_defaults.yml`, so I think it's improbable we run into a case where it's undefined. We can keep as blank 👍